app/notes.py

```diff
@@ -24,11 +24,12 @@ def update_notes():
     def extract_notes_from_html(html: str) -> List[str]:
         soup = BeautifulSoup(html, 'html.parser')
         note_tables = soup.select('table table')
-        quotes = []
+        quotes = {}
         for note in note_tables:
             quote_element = note.select('td:nth-of-type(2) span')[0]
-            text = quote_element.contents[0]
-            quotes.append(text)
+            quote_text = quote_element.contents[0]
+            quote_chapter = quote_element.find_previous('h2').contents[0].contents[0]
+            quotes[quote_text] = quote_chapter
```

Comment:
I think we should create dictionaries for every note, but keep `quotes` a list. The resulting structure added to `NOTES_STORE` would then be:
```
[
    {
        'text': 'the quote text here', 
        'chapter': 'the chapter'
    },
    {
        'text': 'another note and its text here', 
        'chapter': 'another chapter'
    },
]
```
Reasoning: The structure is clearer, and we probably never need to lookup a chapter of a note separately, we just expose both in the API and the frontend.

Reply:
That's actually the same structure I was thinking about doing after I submitted the request if you see my post above. I'll go ahead and do that

Reply:
Ah true, read that some hours before I answered and forgot it 😊